config.yaml

```
@@ -57,20 +57,28 @@ params:
   about:
     enable: true
     title: About Us
-    subtitle: This is the community of Python programmers in Pune and beyond.
+    subtitle: This is the community of **Python programmers** and **Free and Open-source (FOSS) lovers** in Pune and beyond.
```

```suggestion
    subtitle: This is the community of **Python programmers** and **Free/Libre and Open Source Software (FLOSS) lovers** in Pune and beyond.
```
https://www.gnu.org/philosophy/open-source-misses-the-point.html section `“FLOSS” and “FOSS”`